parcels/field.py

```diff
@@ -50,6 +50,8 @@ class Field(object):
            (i.e. beyond the last available time snapshot)
     :param time_periodic: boolean whether to loop periodically over the time component of the Field
            This flag overrides the allow_time_interpolation and sets it to False
+    :param netcdf_engine: engine to use for netcdf reading in xarray. Default is 'netcdf',
+           but in cases where this doesn't work, setting netcdf_engine='scipy' could help
```

Comment:
Why is this doc here in `Field` constructor? Shouldn't it be in `Field.from_netcdf()` and same for `FieldSet.from_netcdf()`?